src/pages/index.html

```
@@ -1,18 +1,16 @@
-<!DOCTYPE html>
+<!doctype html>
 <html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)